app/controllers/users_controller.rb

```diff
@@ -2,22 +2,16 @@ class UsersController < ApplicationController
   before_filter :authenticate_user!
 
   def index
-    @search = User.ransack(params[:q])
-    @search.sorts = 'members_member_uid asc' if @search.sorts.empty?
-
-    @users = @search
-      .result(distinct: false)
-      .joins(members: :account)
-      .eager_load(members: :account)
-      .where(members: { organization: current_organization.id })
-      .page(params[:page])
-      .per(25)
-
-    @memberships = current_organization.members.
-      where(user_id: @users.map(&:id)).
-      includes(:account).each_with_object({}) do |mem, ob|
-        ob[mem.user_id] = mem
-      end
+    default_sort
+
+    @search =
+      current_organization.members.joins(:account, :user).ransack(params[:q])
```

Comment:
There doesn't seem to be any difference between doing `joins` before `ransack.result` or after it, and I don't have a clear reason for it to be in any of them instead of the other. The only "intuition" I have is that by joining after the search, we may risk joining twice. Any input?

Maybe I should just ditch the joins and use only `eager_load`.